src/PKSim.Core/Model/AggregationFunctions.cs

```diff
@@ -28,40 +28,43 @@ public static class AggregationFunctions
 
       private static Aggregate<BoxWhiskerYValue> boxWhiskerAggregation(double percentile)
       {
+         const float outlierRange = 1.5F;
+
          return new Aggregate<double, BoxWhiskerYValue>
          {
             Aggregation = doubles =>
```

Comment:
This was where I tried to make it happen first. But the (doubles) passed as parameter only contains a subset of all values (already pivoted so to speak)